### PR TITLE
PL-1635 : Boilerplate GraphQL Support

### PR DESCRIPTION
**Pull request by BMalaichik**
- added required graphql dependencies
  - added db pg dependenicees
  - updated default config
  - added graphql helper functions
  - added graphql start script & graphiql server launch

@jifeon 
I pointed base branch as `master`.
Please, create branch for merge with proper name
I suggest smth like `master-graphql-support` or `master-graphql`

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+// const path = require('path');
+
 module.exports = {
   app: {
     port: process.env.PORT || 3000,
     suppressErrorStack: process.env.SUPPRESS_ERROR_STACK || false,
   },
 
+  db: {
+    user: 'plyo_mailer',
```
Comment (jifeon) [inline config/default.js]:
```suggestion
    user: 'plyo_boilerplate',
```

would be nice to have abstract name here, so developer must to change it to make code working

File: package.json [hunk 2]
```diff
@@ -12,10 +12,16 @@
     "config": "^2.0.1",
     "cookie-parser": "^1.4.3",
     "express": "^4.16.3",
+    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0",
```
Comment (jifeon) [inline package.json]: can't we leave only latest version?

File: package.json [hunk 3]
```diff
@@ -26,6 +32,7 @@
     "lint-staged": "lint --staged",
     "precommit": "lint --staged",
     "start": "nodemon --watch src src/server.js",
+    "start-server": "DEBUG='postgraphile:graphql,postgraphile:request,postgraphile:postgres*' nodemon --watch src --ignore '*.example.json' --ignore '*.html' --ignore '*.mjml' src/server.js",
```
Comment (jifeon) [inline package.json]: mjml is mailing related stuff, as well as example.json and html

**Review by jifeon — CHANGES_REQUESTED** [review]
Looks good! Please minor stuff and rebase to origin/master_graphql

File: config/default.js (hunk 1 above)
Comment (BMalaichik) [reply config/default.js]: @jifeon uh, accidentally added it

**Review by VlVl — APPROVED** [review]

File: config/default.js [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
 module.exports = {
   app: {
     port: process.env.PORT || 3000,
     suppressErrorStack: process.env.SUPPRESS_ERROR_STACK || false,
   },
   db: {
-    user: '',
-    host: 'db',
+    user: 'plyo_boilerplate',
+    host: 'localhost',
```
Comment (jifeon) [inline config/default.js]:
```suggestion
    host: 'db',
```

We usually run it in docker locally, localhost will not be resolved correctly

**Review by jifeon — APPROVED** [review]
Looks cool!

**Review by jifeon — APPROVED** [review]

**Pull request merged**